gpbft/gpbft.go

```diff
@@ -783,6 +806,13 @@ func (i *instance) tryCommit(round uint64) error {
 		}
 		i.beginNextRound()
 	}
+
+	if timedOut {
```

Comment:
Ditto this isn't true until receiving from a strong quorum, and this should be mutually exclusive with beginNextRound.